src/js/server/utils.js

```diff
@@ -69,6 +78,21 @@ fluid.defaults("gpii.express.user.utils", {
     }
 });
 
+/**
+ * Creates a new gpii-express-user in the configured user database.
+ * `userData` is an object of values to create the new user with.
+ * At the very least this should consist of values for `username`,
+ * `email`, and `password`. The transformation acting on this data
+ * is located at `{gpii.express.user.utils}.rules.createUserWrite`.
+ *
+ * @param {gpii.express.user.utils} that - Utils Component
+ * @param {Object} userData - Object with values for the new user.
+ * @param {String} userData.username - New users username
+ * @param {String} userData.email - New users email
+ * @param {String} userData.password - Password for new account
+ * @return {Promise} Promise container either the new Couch Record for
```

Comment:
I'd suggest lowercasing "record" here.  Also, I think "CouchDB" is probably better than shortening to "Couch".